components/server/src/workspace/workspace-starter.ts

```diff
@@ -614,7 +614,7 @@ export class WorkspaceStarter {
 
         const contextUrlEnv = new EnvironmentVariable();
         contextUrlEnv.setName('GITPOD_WORKSPACE_CONTEXT_URL');
-        contextUrlEnv.setValue(workspace.contextURL);
+        contextUrlEnv.setValue(workspace.context.normalizedContextURL!);
```

Comment:
To make sure this is backwards compatible: `workspace.context.normalizedContextURL || workspace.contextURL`

Reply:
Note that this is a user-facing change: `GITPOD_WORKSPACE_CONTEXT_URL` ends up in each workspace's environment. Still I think it's ok to "break" that implicit contract here given that `supervisor` and IDE's respectively are fine with that. I assume you verified that?

Reply:
yeah it's fine on the supervisor and IDE part